src/middle-end/auth/selectors.js

```diff
@@ -12,7 +12,7 @@ exports.getIsAuthenticated = ({ auth }) => {
     // instead of checking for the payload shape
     const isLoginPayload = (payload) => {
 
-        return ['email', 'password'].every((key) => Object.keys(payload).includes(key));
+        return !!payload.token || ['email', 'password'].every((key) => Object.keys(payload).includes(key));
```

Comment:
I wonder if one of these would be more consistent in style:
```suggestion
        return !!(payload.token || (payload.username && payload.password));
```
```suggestion
        return  Object.keys(payload).includes('token') || ['email', 'password'].every((key) => Object.keys(payload).includes(key));
```

Reply:
The former above might not be quite right: username, password, or token all may be falsy but present.  So `keys.includes('x')` or `payload.hasOwnProperty('x')` or `typeof payload.x !== 'undefined` might be good directions to go.